### PR TITLE
print statements replaced with logging

### LINKED ISSUE #3: Move to use python logger
Currently, all the output uses `print(...)`. I would like to move then to use `logger` class provided by python which can allow for different levels such as DEBUG, INFO etc...

Example of places where print should be replaced:
https://github.com/benwoo1110/testcase-maker/blob/ba1e18bcaa0c2aa11af6f2d37fecb6d2d64120ec/testcase_maker/generator.py#L77

### PR DESCRIPTION
**Pull request by gursi26**
Print statements replaced with logs using Pythons in-built logging module.

### CODE REVIEW

**Review by benwoo1110 — COMMENTED** [review]
Just a few changes I want to the logging configurations.

File: testcase_maker/generator.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 if TYPE_CHECKING:
     from testcase_maker.values import ValueGroup
 
+import logging 
+logging.basicConfig(filename='generator.log', level=logging.DEBUG, format='%(asctime)s %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p')
```
Comment (benwoo1110) [inline testcase_maker/generator.py]: A few things ids prefer for the logging.

1. I think the default level is better to be INFO.
2. Showing log to stdout should be good enough, don't need to save to file.

Comment (gursi26) [reply testcase_maker/generator.py]: Alright, ill make the changes.

**Comment by gursi26** [comment]
Alright, I have made the changes.

**Review by benwoo1110 — APPROVED** [review]
👍

**Pull request merged**